src/services/ui/src/features/package-actions/lib/schemaSwitch.ts

```diff
@@ -33,17 +31,17 @@ const respondToRaiFor: SchemaGroup = {
 };
 
 const enableRaiWithdrawFor: SchemaGroup = {
-  "CHIP SPA": defaultEnableRaiWithdrawSchema,
-  "Medicaid SPA": defaultEnableRaiWithdrawSchema,
-  "1915(b)": defaultEnableRaiWithdrawSchema,
-  "1915(c)": defaultEnableRaiWithdrawSchema,
```

Comment:
I removed these because it didn't make sense to import a single `null`. In the future if we ever want to validate these forms, we can create a new import and export it here.